docs/antora/modules/ROOT/pages/Library_Dependencies.adoc

```diff
@@ -150,12 +150,26 @@ def compileIvyDeps = Agg(
 )
 ----
 
-It is also possible to use higher version of the same library dependencies already  defined in `ivyDeps`, to ensure you compile against a minimal API version, but actually run with the latest available version.
+It is also possible to use higher version of the same library dependencies already defined in `ivyDeps`, to ensure you compile against a minimal API version, but actually run with the latest available version.
+
+== Detecting transitive dependencies
+
+To render a tree of dependencies (transitive included) and possible upgrades for a given module, you can run `mill myModule.ivyDepsTree`.
```

Comment:
I believe this is worth including, as I only saw it in https://stackoverflow.com/questions/58099046/mill-build-tool-how-to-use-evicted yet it's a very helpful command 👍

Reply:
Thanks for adding the mill wrapper to the repo!
I tried using it with Git Bash terminal on Windows 11 (I don't have WSL), but unfortunately it is rendering for this project like:
```
?? ch.epfl.scala:bsp4j:2.1.0-M3
?  ?? org.eclipse.lsp4j:org.eclipse.lsp4j.generator:0.12.0
?  ?  ?? org.eclipse.lsp4j:org.eclipse.lsp4j.jsonrpc:0.12.0
?  ?  ?  ?? com.google.code.gson:gson:2.9.1
?  ?  ?? org.eclipse.xtend:org.eclipse.xtend.lib:2.24.0
?  ?     ?? org.eclipse.xtend:org.eclipse.xtend.lib.macro:2.24.0
```
I can try it from a Mac another day. If you want to get the PR merged sooner feel free to suggest the commit

Reply:
My Mac can't find the adoc file weirdly, so I added it via a code suggestion